content/apply.js

```diff
@@ -270,6 +270,9 @@ const APPLY = (() => {
   }
 
   function fetchParentDomain() {
+    if (/-extension/.test(location.protocol)) {
```

Comment:
>This prevents L276 from working.

Alright, so we resolve the promise?

>This will break if we embed an extension page on a non-extension page. 

I don't usually worry about problems that don't exist.

I guess I should've re-used `IS_OWN_PAGE` which is pretty much the same exact check. 

>this logic should be moved

Not really relevant to this PR.